src/webui/webapplication.cpp

```diff
@@ -367,6 +367,8 @@ void WebApplication::action_command_download()
     CHECK_URI(0);
     QString urls = request().posts["urls"];
     QStringList list = urls.split('\n');
+    bool skipChecking = request().posts["skip_checking"].trimmed() == "true";
+    bool addPaused = request().posts["paused"].trimmed() == "true";
```

Comment:
>But then, `RequestParser::parseContent()` is another issue.

What is the bug here?
If you mean `m_request.posts[pair.first.toLower()] = pair.second;` I think it's not wrong. As @buinsky said:
>Values can be case sensitive